trunk/SUAVE/Optimization/Package_Setups/scipy_setup.py

```diff
@@ -97,3 +107,185 @@ def SciPy_Problem(problem,x):
 
     
     return obj
+
+
+
+def pso(func, lb, ub, ieqcons=[], f_ieqcons=None, args=(), kwargs={}, 
```

Comment:
I think this should be removed from the scipy_setup. It appears to be a standalone routine

Reply:
Ok I moved it to its stand-alone function and added in import